build_tools/kickstarts/rhel-7-cdk-vagrant.ks

```diff
@@ -198,17 +198,23 @@ sleep 120
 # Modify this as needed based on what you want to pre-pull
 # The tag renaming may not be strictly needed but having an image name
 # that points to an internal hostname will likely confuse people
+OPENSHIFT_TAG="v3.2.0.20"
 
-docker pull openshift3/ose:v3.1.1.6
-docker pull openshift3/ose-haproxy-router:v3.1.1.6
-docker pull openshift3/ose-deployer:v3.1.1.6
-docker pull openshift3/ose-docker-registry:v3.1.1.6
-docker pull openshift3/ose-sti-builder:v3.1.1.6
+docker pull openshift3/ose:$OPENSHIFT_TAG
+docker pull openshift3/ose-haproxy-router:$OPENSHIFT_TAG
+docker pull openshift3/ose-deployer:$OPENSHIFT_TAG
+docker pull openshift3/ose-docker-registry:$OPENSHIFT_TAG
+docker pull openshift3/ose-sti-builder:$OPENSHIFT_TAG
 
 echo "Finished pull, running docker images for log debugging"
 docker images
 kill -9 $DOCKER_PID
 
+# Edit sccli.py and openshift_options as per openshift_version
+OPENSHIFT_SED_TAG=`echo $OPENSHIFT_TAG | sed -e 's|\/|\\\/|g' -e 's|\.|\\\\.|g'`
+sed -i -e "s|^IMAGE_TAG = \"latest\"|IMAGE_TAG = \"${OPENSHIFT_SED_TAG}\"|" /opt/adb/sccli.py
+sed -i -e "s|:latest|:${OPENSHIFT_SED_TAG}|" /etc/sysconfig/openshift_option
+
```

Comment:
I am not sure if this is the right thing to do. Need more time to find out if we can some alternative.